### DISCUSSION TITLE
Reolink Duo 3 16MP camera

### DISCUSSION THREAD

**Discussion by lsmith77** [geerlingguy/pi-nvr#15] [General]
Any reason why this 16MP wide angle camera would not work?

https://reolink.com/product/reolink-duo-3-poe/

Looking to buy it and use it for sports video and would prefer to record to a hard drive than fiddle with the SD card slot.

**Comment by geerlingguy** [comment]
(Migrated to Discussions). I'm not sure, but this thread has some good discussion: https://github.com/blakeblackshear/frigate/discussions/4103

I would also love to test their PoE Doorbell cam!
